ros2_ws/src/warehouse_robot/urdf/robot_parts/piston.xacro

```diff
@@ -2,8 +2,8 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
 
     <xacro:macro name="piston" params="prefix">
-        <joint name="${prefix}_joint" type="prismatic">
-            <parent link="base_link" />
+        <joint name="joint_${prefix}" type="prismatic">
```

Comment:
Could you update the `config/my_controllers.yaml` file to have this matching joint name for the piston now? There might be a few other URDF files that need updated with this name

Reply:
So I changed the joint name in a new commit, not sure how to test it though.

Reply:
Thanks! You could try this command with the simulation running. It should get the piston to move up
```
ros2 topic pub -1 /joint_piston/commands std_msgs/msg/Float64MultiArray "data:
> - 1.0"
```